### PR TITLE
🧹 Cleanup if not completely correctify on tap controller

### PR DESCRIPTION
**Pull request by allenhumphreys**

### CODE REVIEW

File: Sources/App/Controllers/OnTap/OnTapController.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+private let iPadDeviceID = "21F17F00-A94B-4BE9-A03D-BA6A20D709FC"
+
+final class OnTapController {
+
+    let slack: Slack
+    let log: Logger
+
+    init(slackClient: Slack, logger: Logger) {
+        self.slack = slackClient
+        self.log = logger
```
Comment (allenhumphreys) [inline Sources/App/Controllers/OnTap/OnTapController.swift]: Notice my use of the logger

Comment (jwilson-slalom) [reply Sources/App/Controllers/OnTap/OnTapController.swift]: Nice

File: Sources/App/Controllers/OnTap/OnTapController.swift [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        do {
+            tap = try request.parameters.next(Tap.self)
+        } catch let error as RoutingError {
+            log.error("url path parameter decoding failed \(error)")
+            return request.response().encode(status: .badRequest, for: request)
+        }
+
+        log.debug("Received \(String(describing: beer)) for tap \(tap)")
+
+        if OnTapMemory.set(beer: beer, on: tap) {
```
Comment (jwilson-slalom) [inline Sources/App/Controllers/OnTap/OnTapController.swift]: Is there a reason we are not actually storing these ? What is the harm in doing so ?

Comment (allenhumphreys) [reply Sources/App/Controllers/OnTap/OnTapController.swift]: I just want to give complete thoughtfulness to the data model design for the _onTap system. I'm not sure how generic I should make it, etc.

Comment (jwilson-slalom) [reply Sources/App/Controllers/OnTap/OnTapController.swift]: Thats fine, just as long as we fix it at some point !

**Pull request merged**